source/Libraries/Iciclecreek.Bot.Builder.Dialogs.Recognizers.Lucene/LuceneDynamicListRecognizer.cs

```diff
@@ -29,7 +29,7 @@ namespace Iciclecreek.Bot.Builder.Dialogs.Recognizers.Lucene
     /// Recognizer implementation which uses Lucene TokenAnalyzer to recognize DynamicLists client side. 
     /// This supports fuzzy "sounds-like" matching using Beider-Morse techniques.
     /// </summary>
-    public class DynamicListRecognizer : Recognizer
+    public class LuceneDynamicListRecognizer : Recognizer
```

Comment:
>LuceneDynamicListRecognizer [](http://example.com/codeflow?start=17&length=27)

You didn't have to rename these.  Because of the namespace on both cs and the $kind the names are unique.